scripts/sync_users.py

```diff
@@ -0,0 +1,60 @@
+# Using requests, get the json file
+import os
+import sys
+import django
+import re
+
+import requests
+from django.contrib.auth import get_user_model
+
+url = "https://api-prod.andela.com/api/v1/users"
+
+headers = {'Authorization': os.getenv('TOKEN')}
```

Comment:
Looks like [AIS](https://github.com/andela/engineering-playbook/wiki/Andela-API-HowTo#using-your-browser) uses `Bearer` not `TOKEN`